tests/contrib/psycopg/test_psycopg.py

```diff
@@ -410,7 +410,7 @@ def test_postgres_dbm_propagation_comment(self):
         cursor.__wrapped__ = mock.Mock()
         cursor.execute("select 'blah'")
         cursor.executemany("select %s", (("foo",), ("bar",)))
-        dbm_comment = " /*dddbs='postgres',dde='staging',ddps='orders-app',ddpv='v7343437-d7ac743'*/"
+        dbm_comment = "/*dddbs='postgres',dde='staging',ddps='orders-app',ddpv='v7343437-d7ac743'*/ "
```

Comment:
Shouldn't this change affect more tests? Are we only checking comments in like 4 places?